public/locales/en/gamepage.json

```
@@ -123,10 +123,12 @@
     },
     "submenu": {
         "addShortcut": "Add shortcut",
+        "addToSteam": "Add to steam",
```

```suggestion
        "addToSteam": "Add to Steam",
```

But I think it should be changed in all files.